src/mlpack/methods/random_forest/CMakeLists.txt

```
@@ -21,3 +21,10 @@ add_julia_binding(random_forest)
 add_go_binding(random_forest)
 add_r_binding(random_forest)
 add_markdown_docs(random_forest "cli;python;julia;go;r" "classification")
+add_cli_executable(extra_trees)
```

```suggestion

add_cli_executable(extra_trees)
```
It's probably a good idea to add just a little bit of extra space here for readability. :+1: